package.json

```diff
@@ -21,7 +21,8 @@
   },
   "main": "./dist/video-js/video.js",
   "dependencies": {
-    "videojs-swf": "4.5.2"
+    "videojs-swf": "4.5.2",
+    "vtt.js": "git+https://github.com/gkatsev/vtt.js.git#custom-build"
```

Comment:
Well honestly I don¹t know, and I don¹t understand which is why I couldn¹t
get anything from your website.

The instructions I got from the creator of the theme were to install your
plugin and then to put:

[videojs mp4="http://longfarms.ca/long-content/uploads/birds_snow.mp4"
preload="auto" autoplay="true" loop="true" controls="false" height="330"
width="960" alt="video of birds with snow falling"]

Into GP Hooks section After Header.

If there is more than that then I guess this is the information I am
lacking.  Would really appreciate it if you could tell me what I¹m missing.

Thank you.







> In package.json 
> <https://github.com/videojs/video.js/pull/1749#discussion-diff-22498269> :
>> > @@ -21,7 +21,8 @@
>> >    },
>> >    "main": "./dist/video-js/video.js",
>> >    "dependencies": {
>> > -    "videojs-swf": "4.5.2"
>> > +    "videojs-swf": "4.5.2",
>> > +    "vtt.js": "git+https://github.com/gkatsev/vtt.js.git#custom-build"
> How is this going to get pulled into video.js?
> 
> ‹
> Reply to this email directly or view it on GitHub
> <https://github.com/videojs/video.js/pull/1749/files#r22498269> .
> 
>  
>    
>  
>